### ISSUE TITLE
parallel="tests" runs all tests in one thread (not in parallel)

### ISSUE THREAD

**Issue by saboteurserg** [cbeust/testng#1662]
### TestNG Version
> Note: only the latest version is supported
6.13
6.13.1
(6.11 - everything is ok)

### Expected behavior
tests should run in parallel

### Actual behavior
all tests are running in one thread, one by one

### Is the issue reproductible on runner?

- [ x ] Maven
- [ x ] IntelliJ

### Test case sample
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite verbose="0" name="Suite1" parallel="tests" thread-count="20">

    <test name="Test1" >
        <classes>
            <class name="test1.Test1" />
        </classes>
    </test>


    <test name="Test2">
        <classes>
            <class name="test1.Test2" />
        </classes>
    </test>


</suite>
```

```
public class Test1 {

      @Test(groups = {"test_testng.Test1.google1"})
    public void google1() {
        System.out.println("THREAD: " + Thread.currentThread().getId() + "");
    }

   @Test()
    public void google2() {
        System.out.println("THREAD: " + Thread.currentThread().getId() + "");
    }


@Test()
    public void google3() {
        System.out.println("THREAD: " + Thread.currentThread().getId() + "");
    }

}

```
```
public class Test2 {

     @Test(groups = {"test_testng.Test2.google1"})
    public void google1() {
        System.out.println("THREAD: " + Thread.currentThread().getId() + "");
    }


  @Test()
    public void google2() {
        System.out.println("THREAD: " + Thread.currentThread().getId() + "");
    }


 @Test()
    public void google3() {
        System.out.println("THREAD: " + Thread.currentThread().getId() + "");
    }

}

```

**Comment by krmahadevan** [comment]
@saboteurserg - There was a similar issue https://github.com/cbeust/testng/issues/1636 which got recently fixed. Can you please try using TestNG 6.14-SNAPSHOT and let us know if your problem is resolved ?

**Comment by saboteurserg** [comment]
@krmahadevan with 6.14-SNAPSHOT it works.

**Comment by juherr** [comment]
Duplicate of #1636
